libs/logger/logger_spdlog.cpp

```diff
@@ -52,9 +52,9 @@ namespace logger {
     static LogPatterns default_patterns;
     if (not is_initialized.test_and_set()) {
       default_patterns.setPattern(
-          LogLevel::kTrace, R"([%Y-%m-%d %H:%M:%S.%F] [th:%t] [%5l] [%n]: %v)");
+          LogLevel::kTrace, R"([%Y-%m-%d %H:%M:%S.%F][th:%t][%=7l][%n]: %v)");
```

Comment:
```suggestion
          LogLevel::kTrace, R"([%Y-%m-%d %H:%M:%S.%F][th:%t][%=8l][%n]: %v)");
```
Because `critical` is 8 characters.